### PR TITLE
Redaction edits

### PR DESCRIPTION
**Pull request by robstradling**

### CODE REVIEW

**Comment by eranmes** [comment]
LGTM, thanks for cleaning up the description!

**Comment by AGWA** [comment]
These are good changes. LGTM too.

File: rfc6962-bis.xml [hunk 1]
```diff
@@ -324,7 +324,15 @@ d0 d1   d2 d3           d0 d1   d2 d3  d4 d5</artwork>
...
             </t>
             <t>
-              <spanx style="verb">SignedData.encapContentInfo.eContent</spanx> MUST contain a TBSCertificate <xref target="RFC5280"/>, which MAY redact certain domain name labels that will be present in the issued certificate (see <xref target="redacting_labels"/>) and MUST NOT contain any SCTs, but which will be otherwise identical to the TBSCertificate in the issued certificate.
+              <spanx style="verb">SignedData.encapContentInfo.eContent</spanx> MUST contain a TBSCertificate <xref target="RFC5280"/> that will be identical to the TBSCertificate in the issued certificate, except that:
+              <list style="symbols">
+                <t>
+                  the <xref target="x509v3_transinfo_extension">Transparency Information</xref> extension MUST be omitted.
+                </t>
+                <t>
+                  the <xref target="RFC5280">subjectAltName</xref> extension MAY be omitted when the CA is using <xref target="domain_name_redaction">domain name redaction</xref>.
```
Comment (benlaurie) [inline rfc6962-bis.xml]: Presumably it MUST be omitted?

File: rfc6962-bis.xml [hunk 2]
```diff
@@ -1439,15 +1463,15 @@ but it is expected there will be a variety.
...
             <list style="symbols">
               <t>The two identities are identical; or,</t>
-              <t>Both identities are DNS names, have the same number of labels, and each label in the SAN identity has a matching label at the same position in the Redacted SAN identity. Two DNS labels are matching if either:
+              <t>Both identities are DNS names, have the same number of labels, and each label in the subjectAltName identity has a matching label at the same position in the redactedSubjectAltName identity. Two DNS labels are matching if either:
                 <list style="symbols">
                   <t>The two labels are identical; or,</t>
-                  <t>The label from the the Redacted SAN identity is equal to REDACTED(label from SAN identity) (<xref target="redacting_labels"/>)</t>
+                  <t>The label from the redactedSubjectAltName identity is equal to REDACT(label from subjectAltName identity) (<xref target="redacting_labels"/>)</t>
```
Comment (benlaurie) [inline rfc6962-bis.xml]: Need to exclude wildcards.

File: rfc6962-bis.xml (hunk 1 above)
Comment (robstradling) [inline rfc6962-bis.xml]: Addressed by https://github.com/google/certificate-transparency-rfcs/pull/179/commits/7afb8c04d7248bb281d399242a77e439ad345b2c

File: rfc6962-bis.xml (hunk 2 above)
Comment (robstradling) [inline rfc6962-bis.xml]: Addressed by https://github.com/google/certificate-transparency-rfcs/pull/179/commits/3de973c5cbd609666e719dac8c04caaff7207fdf

**Pull request merged**